src/Google/autoload.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+$file = __DIR__ . '/../../vendor/autoload.php';
```

Comment:
Ah, OK. I misunderstand that `src/Google/autoload.php` is installed when you're using composer.